### PR TITLE
MWPW-135376 Blog Bulk Update Images

### PR DESCRIPTION
**Pull request by Brandon32**
* Add Images to figure migration
* Update migration reporting

Resolves: [MWPW-135376](https://jira.corp.adobe.com/browse/MWPW-135376)

Test URLs:
* Before: https://main--business-website--adobe.hlx.page/blog/perspectives/10-email-hacks-that-will-save-your-sanity-state-of-enterprise-work-report-2017-2018
* After: https://main--bacom-blog--adobecom.hlx.page/drafts/bulk-update/MWPW-135376-Images/blog/perspectives/10-email-hacks-that-will-save-your-sanity-state-of-enterprise-work-report-2017-2018

### CODE REVIEW

**Review by meganthecoder — APPROVED** [review]

File: bacom-blog/figure/images-to-figure.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ *
+ * @param {object} mdast - mdast tree
+ * @returns {object} mdast tree
+ */
+export async function imageToFigure(mdast) {
+  selectAllBlocks(mdast, 'Images').forEach((imageBlock) => {
+    select('text', imageBlock).value = 'Figure';
+  });
+
+  return mdast;
```
Comment (JasonHowellSlavin) [inline bacom-blog/figure/images-to-figure.js:15]: small detail here, but this function made me realize that for the most part we are mutating the tree, not making copies of it. Did we need to return mdast if we are not saving it and passing it to another function? Maybe we return true?

File: bacom-blog/figure/images-to-figure.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { selectAllBlocks } from '../../utils/mdast-utils.js';
+import { select } from 'unist-util-select'
```
Comment (JasonHowellSlavin) [inline bacom-blog/figure/images-to-figure.js:2]: Lets write a ticket to see what we can refactor using this library after the migration. We may not get to it, but there will be different styles of scripts that we may want to refactor into similar patterns later on.

**Review by JasonHowellSlavin — APPROVED** [review]
Awesome work!

File: bacom-blog/figure/images-to-figure.js (hunk 1 above)
Comment (Brandon32) [reply bacom-blog/figure/images-to-figure.js:15]: It might be useful to return some kind of report or true/false if we were able to migrate without issue.

**Pull request merged**